### DISCUSSION TITLE
difficulty setting up a local development environment

### DISCUSSION THREAD

**Discussion by danish-mehmood** [outline/outline#5094] [General]
when i set NODE_ENV = development in my .env file 
the slack redirect uri has a protocol "HTTP" and slack redirect uri only allows "https"
and when I set NODE_ENV = production  it gives the following error 
ERROR: The server does not support SSL connections
what is it that I am doing wrong ?

**Comment by tommoor** [comment]
A few options:

- Use a provider other than Slack, eg Google does not have the https requirement
- You can generate a local user with: `node build/server/scripts/seed.js email@example.com`
- Setup local [SSL](https://docs.getoutline.com/s/hosting/doc/ssl-pzk7WO8d1n), [mkcert](https://web.dev/how-to-use-local-https/) is a good option for this.

**Reply by baditaflorin** [reply]
> * You can generate a local user with: `node build/server/scripts/seed.js email@example.com`

Wow, thanks for this! I spend hours changing the development setup to allow me to make gmail login work. This is perfect for creating a simple way to login

**Comment by chenyf** [comment]
agree! it's very diffcult to setup local development env.
